src/test/java/com/blogify/blogapi/integration/UserIt.java

```diff
@@ -47,6 +48,9 @@ void client_read_ok() throws ApiException {
     ApiClient client1Client = anApiClient(CLIENT1_TOKEN);
     UserApi api = new UserApi(client1Client);
 
+    User actualUser = api.getUserById(CLIENT1_ID);
+    assertEquals(client1(), actualUser);
```

Comment:
make all apiCall together, and all assert together
so u have to put this near the asserts

Reply:
Okay, thanks!